### PR TITLE
Retail cleanup #12215 keichwa

### PR DESCRIPTION
**Pull request by keichwa**
# Description

General cleanup

# Target branches

Which product version does this PR apply to?

- [ ] Manager-4.0
- [ ] Manager-4.1
- [x] Manager-4.2
- [x] Uyuni

# Links

Fixes https://github.com/SUSE/spacewalk/issues/12215

### CODE REVIEW

File: modules/retail/pages/retail-install-setup.adoc [hunk 1]
```diff
@@ -1,5 +1,7 @@
 [[retail-install-setup]]
-= Setting up
+= Set Up an {smr} Environment
+
+// 2020-08-27, ke: We mustsettle on one term: "POS images" or "terminal images"
```
Comment (keichwa) [inline modules/retail/pages/retail-install-setup.adoc:4]:
```suggestion
// 2020-08-27, ke: We must settle on one term: "POS images" or "terminal images"
```

**Comment by keichwa** [comment]
@aaannz in `retail-mass-config.adoc`, we seem to do mass configuration with branch servers twice.  Can you check it?  See the FIXME statements within the file.

**Review by Loquacity — APPROVED** [review]

**Review by Loquacity — CHANGES_REQUESTED** [review]
Revoking my approval, you have broken xrefs.

**Comment by keichwa** [comment]
> Revoking my approval, you have broken xrefs.

Thanks for spotting.  Additionally I found sth different :)  Please re-tick.

**Review by Loquacity — APPROVED** [review]

**Pull request merged**